### PR TITLE
Kotlin cheat sheet: Fix spelling, indent, empty block

### PR DESCRIPTION
**Pull request by mr-max**
Included suggestions by @moollaza and @GuiltyDolphin from the original PR #2909 

IA: https://duck.co/ia/view/kotlin

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/kotlin.json [hunk 1]
```diff
@@ -1,165 +1,164 @@
...
-      {
-        "key": "val (name, int) = myPerson",
-        "val": "Destructuring classes or collections"
-      }
+    "id":"kotlin_cheat_sheet",
+    "name":"Kotlin",
+    "description":"Basic syntax, functions, and expressions of the Kotlin language",
+    "metadata":{
+        "sourceName":"Kotlin reference",
+        "sourceUrl":"https://kotlinlang.org/docs/reference/"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/kotlin.json]: Lets add a space after colons (e.g., `"metadata": {`).

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kotlin cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/kotlin_mobile.png)

**Comment by moollaza** [comment]
@mr-max Thanks! This LGTM :+1:

@GuiltyDolphin care to sign-off + merge?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kotlin cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/kotlin_mobile.png)

**Comment by GuiltyDolphin** [comment]
@moollaza @mr-max Cool, thanks! :dolphin:

**Pull request merged**